mods/ts/weapons/energyweapons.yaml

```diff
@@ -2,14 +2,40 @@ LtRail:
 	ReloadDelay: 60
 	Range: 6c0
 	Report: BIGGGUN1.AUD
-	Projectile: LaserZap
-		Speed: 1c682
-		BeamWidth: 1
-		BeamDuration: 10
-		Color: 200,0,128,255
+	Projectile: LineImpactProjectile
+		Speed: 5c0
```

Comment:
The projectile speed was much faster in the original game.  Could you speed this up to match?